learn/wavemaker-release-notes/v10-6-3.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "WaveMaker 10.6.3 - Release date: 04 February 2021"
+id: ""
+sidebar_label: "Release v10.6.3"
+---
+*WaveMaker released a new update version with some bug fixes.*
+
+## Improvements
+
+---
+
+- Introduced a new option, `Show At`, in the Prefab UI configuration. When using a Prefab in another application or a Prefab, all Prefab elements' UI properties display by default when you select the Prefab element. Note, when you choose a widget from the `Show At` list, the UI property will display when you select that UI property.
```

Comment:
@swethakundaram  Form layout improvements done by @aparnasj also need to be mentioned here